solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -58,6 +62,67 @@ def apply_validation(obs_df, qfilter, handle_func):
         return validated_obs, counts
 
 
+def apply_fill(fx_data, missing_forecast, start, end, interval_length):
+    """
+    Apply fill procedure to the data from the start to end timestamps.
+
+    Parameters
+    ----------
+    fx_data : pandas.Series or pandas.DataFrame
+        Forecast data with pandas.DatetimeIndex.
+    missing_forecast : str
```

Comment:
> {'drop', 'forward', bool, float}

Not entirely accurate because floats as strings (`'-99.0919'`) are also supported. However, I see how that isn't a big deal. So, updated.

Reply:
> The argument name `missing_forecast` isn't really clear to me. How about `method`? I guess that means we're changing the name along the way from the reports down to this function. Maybe `forecast_fill_method` could be used at all levels of the API?

I was just going off of #423 (and I didn't even copy that correctly). I like `forecast_fill_method`. Updating...

Reply:
Thanks, sorry for the confusion.